### ISSUE TITLE
First run z it prompts not find "C:\users\myname\navidb.csv"

### ISSUE THREAD

**Issue by skywind3000** [JannesMeyer/z.ps#4]

**Comment by JannesMeyer** [comment]
Yes, that makes sense. It looks like when I wrote this a few years ago I assumed everyone would always add it as part of the powershell prompt, rather than hooking into "cd".

May I ask how you set up your "cd" command? Maybe I should add documentation for that to the README.
